f2/src/summary/InformationSummary.js

```diff
@@ -7,22 +7,70 @@ import { useStateValue } from '../utils/state'
 import { testdata, EditButton } from '../ConfirmationSummary'
 import { H2 } from '../components/header'
 import { DescriptionListItem } from '../components/DescriptionListItem'
-import { useIntl } from 'react-intl'
 import { useLingui } from '@lingui/react'
 import { Text } from '../components/text'
 
 export const InformationSummary = props => {
   const [data] = useStateValue()
   const { i18n } = useLingui()
-  const intl = useIntl()
+  const infoReqSummary = []
+  let infoReqLine = " "
```

Comment:
LGTM is complaining about this line assigning a value that isn't used - you can just do `let infoReqLine` instead (without the assignment)

Reply:
fixed